### DISCUSSION TITLE
how to set a env key when the key have many values

### DISCUSSION THREAD

**Discussion by jiangxiaobai** [apache/skywalking#9039] [Q&A] [answered]
# helm install skywalking skywalking/skywalking -n skywalking --set oap.env.SW_OTEL_RECEIVER=default --set oap.env.SW_OTEL_RECEIVER_ENABLED_HANDLERS=oc --set oap.env.SW_OTEL_RECEIVER_ENABLED_OC_RULES="istio-controlplane,k8s-node,k8s-cluster,k8s-service,oap,vm"  -f values-sw-es.yaml
Error: failed parsing --set data: key "k8s-node" has no value (cannot end with ,)

**Comment by kezhenxu94** [comment, marked as answer]
@jiangxiaobai this is a general question of Helm, not SkyWalking, please Google it or ask it in Helm community next time.

For developers, usually when a character is treated with special meaning you can escape it by `\`, and  just a simple Google search can confirm https://www.google.com.hk/search?client=safari&rls=en&q=helm+set+value+with+comma&ie=UTF-8&oe=UTF-8 

https://github.com/helm/helm/issues/1556#issuecomment-393422316

**Reply by jiangxiaobai** [reply]
think you
